jstz_cli/src/account/mod.rs

```diff
@@ -102,29 +124,72 @@ pub fn whoami(cfg: &mut Config) -> Result<()> {
         .as_ref()
         .ok_or(anyhow!("Not logged in!"))?;
 
-    let account = cfg.accounts.get(&alias)?;
+    let (alias, address) =
+        if let Account::Owned(OwnedAccount { alias, address, .. }) =
+            cfg.accounts.get(&alias)?
+        {
+            (alias, address)
+        } else {
+            return Err(anyhow!("Account is not owned"));
+        };
 
-    println!(
-        "Logged in to account {} with address {}",
-        account.alias, account.address
-    );
+    println!("Logged in to account {} with address {}", alias, address);
 
     Ok(())
 }
 
-fn list(cfg: &mut Config) -> Result<()> {
+fn list_accounts(cfg: &mut Config, long: bool) -> Result<()> {
```

Comment:
We should establish a convention of where `cfg` should be (first or last), since currently the CLI uses both (sometimes in the same file 😬 )